src/sentry/features/temporary.py

```diff
@@ -86,6 +86,8 @@ def register_temporary_features(manager: FeatureManager):
     manager.add("organizations:custom-metrics-experimental", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
     # Enables custom metrics extraction rule endpoint
     manager.add("organizations:custom-metrics-extraction-rule", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE)
+    # Enables metrics extrapolation feature
+    manager.add("organizations:metrics-extrapolation", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE)
```

Comment:
This was added to this PR to ensure the project tests pass. When registering the feature, I chose to add it to `features.py` in getsentry based on the guide:

![image](https://github.com/getsentry/sentry/assets/29228205/aed56175-38c5-416d-99a0-a8c461d104a1)